students/stuartkershaw/lesson06/test_mailroom.py

```diff
@@ -0,0 +1,61 @@
+from mailroom import donors
+from mailroom import compose_email
+from mailroom import list_donors
+from mailroom import show_donor_table
+from mailroom import generate_letters
+
+from os import listdir
+from os.path import isfile, join
+
+
+def test_donors():
+    assert isinstance(donors, dict) is True
```

Comment:
This could just be `assert isinstance(donors, dict)` without the `is True`, but this test doesn't seem necessary to me since `donors` should always be a `dict`